cmd/notify-http.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Minio Cloud Storage, (C) 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+type httpNotify struct {
+	Enable bool   `json:"enable"`
+	Addr   string `json:"address"`
+}
+
+func newHTTPNotify(accountID string) (*logrus.Logger, error) {
+	rNotify := serverConfig.GetHTTPNotifyByID(accountID)
+	fmt.Println(rNotify)
+
```

Comment:
Once the minio server has started it is not a problem but it should fail if the config is wrong basically not in 

- proper URL format
- server cannot be resolved i.e DNS resolution fails or "no route to host" etc. 

Errors like network timeout can be ignored i.e the REST endpoint didn't reply back in time. So that minio server can still enable HTTP API endpoint and try connection again later point in time.